### ISSUE TITLE
✅ 🚀 Feature Request: Adding to Flexbox Container options to set:  gap, row-gap, column-gap (Merged)

### ISSUE THREAD

**Issue by datverse** [elementor/elementor#18331]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

There is no easy option to set row/column gap of Flexbox Container when set it wrap.

When set:
`flex-wrap: wrap;`

We CSS set row-gap, column-gap, gap with 2 value for row and column. But in Elementor Flexbox Container we only set gap for all (Elements Gap). 

Current I should set custom css:

```
selector {
    row-gap: 30px;
    column-gap: 40px;
    gap: 40px 50px;
}
```

### Describe the solution you'd like

Adding to Flexbox Container options to set:

gap, row-gap, column-gap

All of this options should have responsive option.

**Comment by fabianenzensperger** [comment]
This change automatically with direction, or?

**Comment by black-eye** [comment]
+1, yes please

**Comment by 79ho3ein** [comment]
+1

**Comment by MichaelDarko** [comment]
+1

**Comment by 3l3mentor** [comment]
+1

**Comment by thejaydip** [comment]
+1

**Comment by tomas-eklund** [comment]
This is absolutely necessary.

**Comment by joehdevwp** [comment]
+1

**Comment by donwattz1459** [comment]
We absolutely need the ability to set default gap and widget spacing. Having to go in and set 10px padding left and right on each container is time consuming especially if we forget while building.

**Comment by joehdevwp** [comment]
> We absolutely need the ability to set default gap and widget spacing. Having to go in and set 10px padding left and right on each container is time consuming especially if we forget while building.

It's much worse than that - imagine if you set 10px padding on every container and then later decide it ought to be 15px, but by that time you've got 30+ pages on your website. It's like being back in the Dark Ages of web design.

And there's no good way (I know of) to target just the parent containers to adding padding via css without adding a class to each one just for that purpose.

**Comment by nicholaszein** [comment]
Hello all,

thank you for joining the discussion! This request is being considered but we do not have an ETA on delivery. The focus is to make the Flexbox Container a stable feature and to improve the overall compatibility with everything else inside the builder.

We might look into implementing this feature in the next phases. But for now, we are unable to prioritize it at this time.

If you decide to take it upon yourself to write your own solution, we would be happy to adopt it after reviewing the code! 😉 

Take care!

**Comment by datverse** [comment]
> Hello all,
> 
> 
> 
> thank you for joining the discussion! This request is being considered but we do not have an ETA on delivery. The focus is to make the Flexbox Container a stable feature and to improve the overall compatibility with everything else inside the builder.
> 
> 
> 
> We might look into implementing this feature in the next phases. But for now, we are unable to prioritize it at this time.
> 
> 
> 
> If you decide to take it upon yourself to write your own solution, we would be happy to adopt it after reviewing the code! 😉 
> 
> 
> 
> Take care!

Thank for your reply. We hope and waiting this for next release. Maybe 3.9 version.

Thanks ❤️

**Comment by fabianenzensperger** [comment]
Okay, i should write here for this subject: +1! 👀

**Comment by datverse** [comment]
Hi @nicholaszein 
We hope and waiting this for next release 3.11.0. 
Can this issue be updated to the upcoming 3.11.0 version? 
Thanks ❤️

**Comment by nicholaszein** [comment]
Hey everyone!

We have great news! 🙌 

Now that Containers have been released as a stable feature that is active by default on new installations, we took the opportunity to also include separate column and row gaps by default on all Containers. Therefore...

### 📢 We're happy to announce that this feature was released with Elementor v3.16! 🥳 

Thank you all for your contribution and for sharing your feedback! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
